gips/data/merra/merra.py

```diff
@@ -24,6 +24,15 @@
 import os
 import datetime
 import time
+
+from gips.data.merra.install_basic_client import install_basic_client
+
+URI = "urs.earthdata.nasa.gov"
+USER = "bobbyhbraswell"
+PASSWD = "Coffeedog_2"
```

Comment:
I think I prefer the latter.  The `.netrc` file is a standard, I think, but keeping everything in GIPS settings seems better to me.